source/includes/_corda.md

```diff
@@ -116,13 +116,13 @@ override fun verify(tx: LedgerTransaction) {
     requireThat {
         ...
 
-        // Check tha Oraclise's answer 
+        // Check tha Oraclise's answer
         val rate = answerCommand.value.rawValue as String
         "The rate USD/GBP must be over $USD_GBP_RATE_THRESH" using (rate.toDouble() > USD_GBP_RATE_THRESH)
 
         // You can use the proof verification tool in the contract as well
         val proofVerificationTool = OraclizeUtils.ProofVerificationTool()
-        "Oraclize's proof verification failed" using  (
```

Comment:
@D-Nice not sure if this is worth the chagne

Reply:
yep dont worry about yet, that will be changed as well, once i have our code rebrand complete as well